.github/workflows/tests.yml

```diff
@@ -65,6 +65,7 @@ jobs:
     - name: Install Playwright Browsers
       run: npx playwright install --with-deps
     - run: npm run playwright
+#    - run: npm run playwright-e2e
```

Comment:
?

Reply:
wanted to follow up with a plan for how/when to run the e2e tests

Reply:
File a follow up task to resolve :)

Reply:
yep, done